maquetado/scss/footer.scss

```diff
@@ -0,0 +1,25 @@
+.footer-project {
+  align-items: center;
+  background-color: $blue;
+  display: flex;
+  flex-direction: column;
+  padding: 2rem 0;
+  position: relative;
+  width: 100%;
+}
+
+.footer-title,
+.footer-copyright {
```

Comment:
try to create `footer-text` class or something like that and use it in the navbar texts